source/docs/yearly-changes/yearly-changelog.rst

```diff
@@ -0,0 +1,128 @@
+   CommandSwerveDrivetrain drivetrain = TunerConstants.DriveTrain; // My drivetrain
+   SwerveRequest.FieldCentric drive = new SwerveRequest.FieldCentric().withIsOpenLoop(true); // I want field-centric
+                                                                                             // driving in open loop
+   SwerveRequest.SwerveDriveBrake brake = new SwerveRequest.SwerveDriveBrake();
+   SwerveRequest.PointWheelsAt point = new SwerveRequest.PointWheelsAt();
+   Telemetry logger = new Telemetry(MaxSpeed);
+
+   private void configureBindings() {
+      drivetrain.setDefaultCommand( // Drivetrain will execute this command periodically
+      drivetrain.applyRequest(() ->
+         drive.withVelocityX(-joystick.getLeftY() * MaxSpeed) // Drive forward with negative Y (forward)
+         .withVelocityY(-joystick.getLeftX() * MaxSpeed) // Drive left with negative X (left)
+         .withRotationalRate(-joystick.getRightX() * MaxAngularRate) // Drive counterclockwise with negative X (left)
+         )
+      );
+
+      joystick.a().whileTrue(drivetrain.applyRequest(() -> brake));
+      joystick.b().whileTrue(drivetrain
+         .applyRequest(() -> point.withModuleDirection(new Rotation2d(-joystick.getLeftY(), -joystick.getLeftX()))));
+
+      if (Utils.isSimulation()) {
+         drivetrain.seedFieldRelative(new Pose2d(new Translation2d(), Rotation2d.fromDegrees(90)));
+      }
+
+      drivetrain.registerTelemetry(logger::telemeterize);
+   }
+
+.. image:: images/swerve-simulation-video.gif
+   :alt: GIF showing swerve simulation support
+
+.. important:: Swerve API requires all necessary swerve devices to be v6 devices. e.g. 4 drive TalonFX, 4 steer TalonFX, 1 Pigeon 2.0, 4 CANcoders.
+
+Signal Logging
+^^^^^^^^^^^^^^
+
+We've added a comprehensive signal logger API (Java, C++, Python) that represents a real-time capture of signals for supported devices. Signal logging can be useful for analysis of signals over a period of time. In applications, they can be useful for tuning PID gains, characterization of systems, analyzing latency on a system and much more.
+
+Information on configuring the logger and extracting the logs are available in the API Reference and Tuner sections.
+
+.. image:: images/tuner-x-log-extractor.png
+   :alt: Log extractor page in Tuner X
+
+Differential Mechanisms
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. important:: Both ``DifferentialMechanism`` (Pro only) and ``SimpleDifferentialMechanism`` require `CANFD <https://store.ctr-electronics.com/canivore/>`__.
+
+``DifferentialMechanism`` (Java, C++, Python) provides an easy way to control two-axis differential mechanisms such as two motor elevators (motor on the left and right side of the elevator but is not mechanically linked). Information on this is available here.
+
+New Configs
+^^^^^^^^^^^
+
+We've added several new configs. A full list of available configs is available in the ``configs`` (Java, C++, Python) namespace.
+
```

Comment:
Other major changes we may want to mention:
- New helper methods for working with multiple devices/StatusSignals
- Improvements to status signal update frequency behavior and getting applied update frequency
- Disabling status signals and optimizeBusUtilization
- Orchestra added, now with support for multiple devices playing a single track
- More feedforwards
- Support for roboRIO motion profiles using Velocity/Acceleration setpoints in Position/Velocity controls
- Improved support for unit tests (I already have a PR open to add a page to docs)
- Support for remote limits and more remote sensors
- New Motion Magic control requests

Reply:
I explicitly didn't include orchestra as that's a port from v5 feature.